### PR TITLE
Adding information layer

### PR DESCRIPTION
**Pull request by ssteinbuss**
Basic version of informatio  layer closing #17

### CODE REVIEW

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+has been developed along the analysis, findings, and requirements of the
+Conceptual Representation. Based on a stack of W3C Semantic Web
+technology standards[^5] and standard modeling vocabularies (DCAT[^6],
+ODRL[^7], SKOS[^8], etc.), it provides a formal, machine-interpretable
+specification of concepts envisaged by the Conceptual Representation,
+residing at the persistent namespace URI <https://w3id.org/idsa/core/>.
+Furthermore, it details and formally defines entities of the
+International Data Spaces in order to be able to share, search for, and
+reason upon the structured metadata describing these entities. As such,
+it comprises a complete referential model allowing the derivation of a
```
Comment (clange) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md:77]:
```suggestion
reason upon the structured metadata describing these entities. The IDS Vocabulary is defined using RDF Schema[^rdfs] and the OWL Web Ontology Language[^owl]; additionally, descriptions of Digital Resources can be validated against SHACL shapes[^shacl] that express syntactic and semantic conditions. Queries against, e.g., Data Resources listed in the Data Catalogue of a Connector or Broker, or against Software Resources available from an App Store, can be formulated in query languages such as SPARQL[^sparql].  Thus, the
Declarative Representation comprises a complete referential model allowing the derivation of a
```
Now also explicitly referencing the basic W3C standards mentioned in the figure.

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+[^4]: https://github.com/International-Data-Spaces-Association/InformationModel
+
+[^5]: https://www.w3.org/standards/semanticweb/
+
+[^6]: https://www.w3.org/TR/vocab-dcat-2/
+
+[^7]: https://www.w3.org/TR/odrl-model/
+
+[^8]: https://www.w3.org/TR/skos-reference/
+
```
Comment (clange) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md:211]:
```suggestion

[^rdfs]: RDF Schema 1.1. W3C Recommendation 25 February 2014. https://www.w3.org/TR/rdf-schema/

[^owl]: OWL 2 Web Ontology Language Document Overview (Second Edition). W3C Recommendation 11 December 2012. https://www.w3.org/TR/owl2-overview/

[^shacl]: Shapes Constraint Language (SHACL). W3C Recommendation 20 July 2017. https://www.w3.org/TR/shacl/

[^sparql]: SPARQL 1.1 Query Language. W3C Recommendation 21 March 2013. https://www.w3.org/TR/sparql11-query/

```
References to the basic W3C standards that we now mention above

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+a look at these alternative implementations.
+
+#### Declarative Representation
+
+The Declarative Representation (IDS Vocabulary) provides a normative
+view of the Information Model of the International Data Spaces.[^4] It
+has been developed along the analysis, findings, and requirements of the
+Conceptual Representation. Based on a stack of W3C Semantic Web
+technology standards[^5] and standard modeling vocabularies (DCAT[^6],
+ODRL[^7], SKOS[^8], etc.), it provides a formal, machine-interpretable
```
Comment (clange) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md:71]:
```suggestion
technology standards[^5] and standard modeling vocabularies (the Data Catalog Vocabulary DCAT[^6],
the Open Digital Rights Language ODRL[^7], the Simple Knowledge Organization System SKOS[^8], etc.), it provides a formal, machine-interpretable
```
expanded the standards acronyms

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+specification of concepts envisaged by the Conceptual Representation,
+residing at the persistent namespace URI <https://w3id.org/idsa/core/>.
```
Comment (clange) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md:73]:
```suggestion
residing at the persistent namespace URI <https://w3id.org/idsa/core/> according to best practices for publishing linked data[^bp-ld].
```

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md (hunk 2 above)
Comment (clange) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md:211]:
```suggestion

[^bp-ld]: Best Practices for Publishing Linked Data. W3C Working Group Note 09 January 2014. http://www.w3.org/TR/ld-bp/

```
References to the basic W3C standards that we now mention above

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+:::
+
+#### Concern Hexagon
+
+To illustrate the main modeling [c]{.underline}oncerns of Digital
+Resources in a way easy to memorize, the mnemonic hexagonal arrangement
+of [c]{.underline}arbon atoms can be used (C-Hexagon), as shown in
+Figure
+[2](#fig:Outline_of_the_ConcernBasic_concern_hexagon){reference-type="ref"
+reference="fig:Outline_of_the_ConcernBasic_concern_hexagon"}. As a
```
Comment (clange) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md:172]:
```suggestion
reference="fig:Outline_of_the_ConcernBasic_concern_hexagon"}.

As a
```
Now spending one paragraph on each concern, reintroducing revised versions of the summaries we had in the RAM 3.0.

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+Resource's content is its most essential aspect, Content is located at
+the top of the hexagon. This content is interpretable by references to a
```
Comment (clange) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md:174]:
```suggestion
the top of the hexagon. The *Content* concern deals with 
1. the description of a Resource's abstract substance, 
2. its serialization as a representation in a machine-interpretable format, and 
3. the materializations of these representations at certain points in time as one or more instances (e.g., values or artifacts).

Content is interpretable by references to a
```
Now spending one paragraph on each concern, reintroducing revised versions of the summaries we had in the RAM 3.0.

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+shared, formally defined Concept, whereas links to a particular
+*C*ontext (in terms of time, place, or real-world entities) make the
+content potentially relevant for certain Data Consumers. So the upper
```
Comment (clange) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md:177]:
```suggestion
*C*ontext (in terms of, e.g.,
* time, 
* place, or 
* real-world entities)
make the
Content potentially relevant for certain Data Consumers.

Thus, the upper
```
Now spending one paragraph on each concern, reintroducing revised versions of the summaries we had in the RAM 3.0.

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+part of the C-Hexagon deals with the what aspects, independently of
+Data Exchange, Data Sharing or Data Utilization. The lower part relates
+to the how aspects; i.e. how the content is exchanged
+(*C*ommunication) and under which conditions (*C*ommodity). The
```
Comment (clange) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md:181]:
```suggestion
(*C*ommunication) and under which conditions (*C*ommodity). 

The *Communication* concern deals with means to communicate a Resource's Content in one of the Representations
available, e.g., 
* by sending messages in some communication protocol
* to a resource or service endpoint or to an IDS Connector
* in order to perform an operation.

The *Commodity* concern helps to address the value and utility of a Resource in terms of, e.g.,
* its provenance,
* its quality, and
* the (usage) policies attached to it, e.g., the obligation to pay a certain price for its consumption.

The 
```
Now spending one paragraph on each concern, reintroducing revised versions of the summaries we had in the RAM 3.0.

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+*C*ommunity of Trust concern refers to the distinctive feature of the
+International Data Spaces being an ecosystem of certified participants
+and components that exchange and share Digital Resources in accordance
+with usage policies ensuring data sovereignty.
```
Comment (clange) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md:185]:
```suggestion
operating certified components, such as Connectors. Using such components, Participants exchange and share Digital Resources in a secure and trusted way in accordance
with contracts composed of usage policies, thus ensuring data sovereignty.
```
Now spending one paragraph on each concern, reintroducing revised versions of the summaries we had in the RAM 3.0.

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/README.md [hunk 10]
```diff
@@ -0,0 +1,3 @@
+# Information layers
```
Comment (clange) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/README.md:1]:
```suggestion
# Information layer
```

File: documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md [hunk 11]
```diff
@@ -0,0 +1,224 @@
...
+
+The level of detail differs across the individual concerns. The
+selection of their constituting aspects may change in light of new
+requirements and insights. Modeling concerns may inform, but do not
```
Comment (clange) [inline documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md:189]:
```suggestion
::: Center
![Detailed Concern Hexagon](./media/image53.png){#fig:Detailed_Concern_Hexagon}
:::

The level of detail differs across the individual concerns. The
selection of their constituting aspects may change in light of new
requirements and insights; Fig. [3](#fig:Detailed_Concern_Hexagon){reference-type="ref"
reference="fig:Detailed_Concern_Hexagon"} suggests one such expansion of the C-Hexagon to one more level of detail.

Modeling concerns may inform, but do not
```
Reinstated the old detailed C-Hexagon figure, now clarifying that this it no gospel but one possible approach.

**Review by clange — APPROVED** [review]
Complete revision. After applying my additional suggestions made here, this should be fine.

**Review by HeinrichPet — APPROVED** [review]

**Pull request merged**